requirements/dev.txt

```diff
@@ -128,7 +132,7 @@ edx-django-utils==3.13.0
     #   super-csv
 edx-i18n-tools==0.5.3
     # via -r requirements/dev.in
-edx-lint==1.6
+edx-lint==3.0.2
```

Comment:
py3.5 support dropped, no worries there https://github.com/edx/edx-lint/blob/master/CHANGELOG.rst#200---2021-01-21
3.0.0 seems like just added features, but again it doesn't break the build, so /shrug https://github.com/edx/edx-lint/blob/master/CHANGELOG.rst#300---2021-01-24